### PR TITLE
IANA mediaType registration review

### PR DESCRIPTION
**Pull request by SteveLasker**
Initially submitted to: : https://www.iana.org/form/media-types
PR used for feedback from IANA

Signed-off-by: Steve Lasker <stevelasker@hotmail.com>

### CODE REVIEW

File: iana-mediatypes/oci-mediatype.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+**Name:** Steve Lasker
+
+**Email:** tob@opencontainers.org
+
+**Media type name:** application
+
+**Media subtype name:** application/vnd.oci.image.manifest.v1+json
```
Comment (SteveLasker) [inline iana-mediatypes/oci-mediatype.md]: > IANA: Given that this is a JSON format - this wasn't clear in the original registration - the recommendation is that the media type have +json suffix. Note that this isn't required, but it is strongly recommended.  

Initially submitted as a root mediatype. Changed from `application/vnd.oci.image.manifest` to `application/vnd.oci.image.manifest.v1+json` to reflect no assumptions are made about parent ownership.
This also accounts for the json extension feedback.

File: iana-mediatypes/oci-mediatype.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+**Required parameters:** n/a
+
+**Optional parameters:** n/a
+
+**Encoding considerations:** binary
+
+**Security Considerations:** This JSON format media type inherits the security considerations for JSON. See RFC 8259 section 12. (https://tools.ietf.org/html/rfc8259#section-12)
```
Comment (SteveLasker) [inline iana-mediatypes/oci-mediatype.md]: > IANA: This JSON format media type inherits the security considerations for JSON. See RFC 7303 section 12.

Changed, but I believe you were referring to RFC 8259, section 12

File: iana-mediatypes/oci-mediatype.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+**Privacy Information:** The manifest contains configuration information, with optional annotations. While users may put any content they wish into these annotations, there are no requirements for privacy information that must be protected.
```
Comment (SteveLasker) [inline iana-mediatypes/oci-mediatype.md]: > IANA: "requirements of" -> "requirements for"?

File: iana-mediatypes/oci-mediatype.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+**Extensibility:** The json payload supports annotations as strings. No executable information or code paths are enabled.
+
+**Compression:** The manifest type is not compressed in normal workflows.
+
+**Links:** The manifest contains digests of other linked content, which are separately retrieved, with their own security considerations.
```
Comment (SteveLasker) [inline iana-mediatypes/oci-mediatype.md]: > IANA: Missing from this list is integrity protection, and it appears to be important: Unless I misunderstand something, tampering with the content of a manifest could cause bad things to happen. As such, a note about the need to protect the integrity of objects of this type would seem to be needed.

Added "Content Digest spec: https://github.com/opencontainers/distribution-spec/blob/master/spec.md#content-digests" to line 29

File: iana-mediatypes/oci-mediatype.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+**Interoperability Considerations:** An OCI image manifest may represent one of many different platforms and architectures. When retrieving a manifest, the config descriptor https://github.com/opencontainers/image-spec/blob/master/config.md returns a an object the consumer can determine if this manifest applies to the host platform and architecture.
+
+**Published specification:**
+application/vnd.oci.image.manifest.v1+json spec: https://github.com/opencontainers/image-spec/blob/master/manifest.md
+Content Digest spec: https://github.com/opencontainers/distribution-spec/blob/master/spec.md#content-digests
```
Comment (SteveLasker) [inline iana-mediatypes/oci-mediatype.md]: > IANA: This reference doesn't appear to be correct. Isn't the correct reference for this specific format

 Fixed link

Comment (mikebrow) [reply iana-mediatypes/oci-mediatype.md:29]: the content digest link isn't needed here..

File: iana-mediatypes/oci-mediatype.md (hunk 3 above)
Comment (mikebrow) [reply iana-mediatypes/oci-mediatype.md:17]: image manifest...

**Review by mikebrow — COMMENTED** [review]
see requested changes

**Comment by SteveLasker** [comment]
Closing as the iana.org registration was completed: https://www.iana.org/assignments/media-types/application/vnd.oci.image.manifest.v1+json

**Pull request closed without merging**